tests/phpunit/Integration/XmlLayoutFileValidityTest.php

```diff
@@ -39,7 +39,7 @@
  * @ingroup Skins
  * @ingroup Test
  */
-class XmlLayoutFileValidityTest extends \PHPUnit\Framework\TestCase {
+class XmlLayoutFileValidityTest extends \MediaWikiTestCase {
```

Comment:
Why would you inherit from `MediaWikiTestCase` when inheriting from `TestCase` is sufficient?